templates/common/gcp/files/usr-local-bin-update-dns-server.yaml

```diff
@@ -0,0 +1,23 @@
+mode: 0755
+path: "/usr/local/bin/update-dns-server"
+contents:
+  inline: |
+    #!/bin/bash
+    
+    echo "Executing update-dns-server"
+
+    mkdir -p /etc/NetworkManager/conf.d
+
+    echo "#Added by OpenShift" > /etc/NetworkManager/conf.d/dns-servers.conf
```

Comment:
This might be easier to read as:

```
cat <<EOF | tee /etc/NetworkManager/conf.d/dns-servers.conf
# Added by OpenShift
[global-dns-domain-*]
servers=$(ip --json route get 8.8.8.8 | jq ".[0].prefsrc"),169.254.169.254
EOF
```

Reply:
Done.